styles/styles.css

```diff
@@ -26,6 +26,7 @@
   --color-blue-dark: #053C6D;
   --color-deep-purple: #1A0D7C;
   --color-white: #FFF;
+  --color-off-white: #fff;
```

Comment:
you can use the existing --color-white

Reply:
there is slight difference between both

Reply:
the color codes look the same.

Reply:
I have experienced slight different but may be I need to get my eye checked :D Both are same.

Reply:
>  I need to get my eye checked :D Both are same.

🤣